include-build/roborazzi-gradle-plugin/src/main/java/io/github/takahirom/roborazzi/GenerateComposePreviewRobolectricTestsExtension.kt

```diff
@@ -327,3 +330,82 @@ private fun verifyLibraryDependencies(
   )
   requiredLibraries.forEach { allDependencies.checkExists(it) }
 }
+
+private fun verifyComposablePreviewScannerVersion(
+  project: Project
```

Comment:
Oh I see!
I thought that the import in the generated code was the problem, but it is in the roborazzi module that provides support for composablePreviewScanner right?

I think that would explain why it is source code compatible but not at bytecode level...

In that case I think it is only fixable by enforcing the user to use 0.7.0+ when using Roborazzi. So this looks fine :)

Reply:
I suspect the issue is that the bytecode for `roborazzi-compose-preview-scanner-support` invokes `AndroidPreviewScreenshotIdBuilder.ignoreClassName`. I'm guessing a `NoSuchMethodError` occurs because that method doesn't exist.